jaeger-core/src/main/java/com/uber/jaeger/Configuration.java

```diff
@@ -67,19 +78,31 @@ public Configuration(
       reporterConfig = new ReporterConfiguration(null, null, null, null, null);
     }
     this.reporterConfig = reporterConfig;
+
+    statsFactory = new StatsFactoryImpl(new NullStatsReporter());
   }
 
-  public Tracer.Builder getTracerBuilder(StatsFactory statsFactory) {
+  public Tracer.Builder getTracerBuilder() {
     Metrics metrics = new Metrics(statsFactory);
     Reporter reporter = reporterConfig.getReporter(metrics);
     Sampler sampler = samplerConfig.createSampler(serviceName, metrics);
     return new Tracer.Builder(serviceName, reporter, sampler).withMetrics(metrics);
   }
 
   public Tracer getNoopTracer() {
-    Reporter reporter = new NoopReporter();
-    Sampler sampler = new ConstSampler(false);
-    return new Tracer.Builder(serviceName, reporter, sampler).build();
+    return Tracer.getNoopTracer(serviceName);
```

Comment:
I'd prefer to keep this in the config. What's the reason for creating this method on the main Tracer?